### ISSUE TITLE
Capture output in env var

### ISSUE THREAD

**Issue by ghaabor** [bitrise-steplib/steps-script-runner#5]

**Comment by viktorbenei** [comment]
You mean the whole output of the step/script, as-it-is? I don't think that's a good idea, quite wasteful to be honest, and without any formatting it will be super hard to parse in another step(s).

A better solution is to export exactly what you want to export, in the format you want to export it in, using [envman](https://github.com/bitrise-io/envman/). Related docs: http://devcenter.bitrise.io/tips-and-tricks/expose-environment-variable/
